### DISCUSSION TITLE
canvas image in not working in frame

### DISCUSSION THREAD

**Discussion by Hancie123** [TomSchimansky/CustomTkinter#595] [Q&A]
frame1 = customtkinter.CTkFrame(master=root, width=400, height=400, corner_radius=20)
        frame1.pack(side=LEFT, padx=20)

        canvas = Canvas(frame1, width=900, height=900, bg="#212325", borderwidth=0, bd=0, highlightthickness=0)
        image=ImageTk.PhotoImage(Image.open("E:\College Assignments\Second Semester\Python\Taxi Booking System\Images\Tablet login-rafiki.png"))
        canvas.create_image(120, 120, image=image)
        canvas.pack()

**Comment by abhaymakes** [comment]
Here, this will work : 
```python
import customtkinter as ctk
from PIL import ImageTk, Image

root = ctk.CTk()

canvas = ctk.CTkCanvas(root, width=900, height=900, bg="#212325", borderwidth=0, bd=0, highlightthickness=0)
image=ImageTk.PhotoImage(Image.open("<your_image_path>"))
canvas.create_image(120, 120, image=image)
canvas.pack()


root.mainloop()

```
